toolkit/config.dev.yaml

```diff
@@ -1,5 +1,4 @@
 environment:
-  name: dev
```

Comment:
are we never use name in other scripts ?

Reply:
maybe not related could not keep it for my self :)

<img width="1582" alt="Scherm­afbeelding 2024-07-24 om 10 15 39" src="https://github.com/user-attachments/assets/d9491a00-6ce9-45a2-bd9a-5e3b7640e031">

Reply:
toolkit has made `name` redundant since it was always supposed to be the same as the name in the file name `config.<NAME>.yaml` and these configs are only used by toolkit cli nothing else 👍🏻

Reply:
🤣 definitely not related, but that's a funny coincidence